src/tests/ftest/performance/ior_easy.yaml

```diff
@@ -90,7 +90,8 @@ ior_mpiio_sx:
   transfer_size: 1MiB
 dfuse:
   mount_dir: "/tmp/daos_dfuse"
-  disable_caching: True
+  disable_caching: true
+  eq-count: 4
```

Comment:
FYI that this particular test isn't covered with the `dfuse` tag, but `performance_dfuse` would cover it. The tag made sense when I added the test, now I don't think so, since obviously you expect it to be covered by `dfuse`